aptos-move/aptos-gas/src/gas_meter.rs

```diff
@@ -27,6 +27,8 @@ use move_vm_types::{
 use std::collections::BTreeMap;
 
 // Change log:
+// - V4
+//   - Consider memory leaked for event natives
```

Comment:
@alinush  it'll be nice if you can add your change at v4 as well

Reply:
Ehh, which change?